packages/labs/ssr/src/test/test-files/render-test-module.ts

```diff
@@ -9,7 +9,7 @@ import {repeat} from 'lit/directives/repeat.js';
 import {classMap} from 'lit/directives/class-map.js';
 import {LitElement, css, PropertyValues} from 'lit';
 import {property, customElement} from 'lit/decorators.js';
-import {serverhtml} from '../../lib/server-template.js';
+import {html} from '../../lib/server-template.js';
```

Comment:
This one should also be `html as serverhtml` as it clashes with the regular `html`.

Reply:
Ah, yep, fixed